README.md

```diff
@@ -141,17 +139,16 @@ luamp_filename: new_year_boardgames_party.mp4
 If you do not need prefix and postfix, you can omit them from the regexp, but do make sure you `set` them to an empty string in the location. Here's the minimal viable example for simpler URLs with no prefix/postfix:
 
 ```
-location ~ ^/(?<luamp_flags>([0-9a-zA-Z_,\.:]+)\/|)(?<luamp_filename>[0-9a-zA-Z_\-\.]+\.mp4)$ {
+location @luamp_media_processor {
```

Comment:
yep. this is works only for the new boilerplate, i.e. images. would you like to separate instruction for videos or it could wait for moving rest of the code to the new boilerplate?